style.css

```diff
@@ -1 +1,254 @@
 /*  Add your styling here */
+
+body {
+  font-family: "Mukta", sans-serif;
+  color: rgb(133, 120, 120);
+  margin: 0;
+}
+
+.index-main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 0.2fr 0.7fr 0.5fr 0.7fr 0.2fr;
+  gap: 2em;
+  grid-template-areas:
+    "hero hero"
+    "description description"
+    "d-picture d-picture"
+    "products products"
+    "footer footer";
+}
+
+.index-hero {
+  grid-area: hero;
+  font-weight: bold;
+  border-bottom: 2px solid rgb(186, 219, 208);
+  padding: 0;
+}
+
+.index-description {
+  border: 2px solid rgb(186, 219, 208);
+  border-radius: 1em;
+  grid-area: description;
+  display: flex;
+  gap: 1em;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0;
+  padding: 2em;
+  margin: 0.5em;
+}
+
+.index-description p,
+h1 {
+  margin: 0;
+}
+
+.index-description-picture {
+  grid-area: d-picture;
+  background-image: url("img/d-image.jpeg");
+  background-size: cover;
+}
+
+.index-products {
+  grid-area: products;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 10px;
+}
+.index-products img {
+  width: 45%;
+  height: auto;
+}
+/*motto*/
+.motto {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.motto img {
+  width: 20%;
+}
+.motto p {
+  font-size: 1.2em;
+  text-align: right;
+  margin-bottom: 0;
+  margin-right: 0;
+}
+
+/*Nav bar*/
+.topnav {
```

Comment:
I've commented on your other work, but generally `top-nav` would be preferable here.